app/index.html

```diff
@@ -24,6 +24,8 @@
   <script type="text/javascript" src="js/livebus/livebus.factory.js"></script>
 
   <script type="text/javascript" src="js/shared/shared.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.directive.js"></script>
```

Comment:
That part is specifically for components (Angular 1.5 feature) which we are not using but below that there's a guide for directives. They seem to prefer to use a prefix for the directive selector name. I'm ok with that but I don't know if that applies for general directives supposed to be shared by various projects? The filenames and modules don't seem to have the app prefix in that style guide.